Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -933,7 +933,7 @@ theorem exists_ratFunc_val_lt (f : K⸨X⸩) (γ : ℤₘ₀ˣ) :
     · erw [hs, ← F_mul, PowerSeries.coe_pow, PowerSeries.coe_X, RatFunc.coe_mul, zpow_neg,
         zpow_ofNat, inv_eq_one_div (RatFunc.X ^ s), RatFunc.coe_div, RatFunc.coe_pow, RatFunc.coe_X,
         RatFunc.coe_one, ← inv_eq_one_div, ← mul_sub, map_mul, map_inv₀, ← PowerSeries.coe_X,
-        valuation_X_pow, ← hs, ← RatFunc.coe_coe, ← coe_sub, ← coe_algebraMap,
+        valuation_X_pow, ← hs, ← RatFunc.coe_coe, ← PowerSeries.coe_sub, ← coe_algebraMap,
```

Comment:
What's going on here? Which `coe_sub` was this before, and is this new one somehow the right one?

Reply:
It would be great to replace this `erw` with a `rw` if possible to clear this up.

Reply:
Yeah I think the `coe_sub` before was `PowerSeries.coe_sub`. Because I have created another declaration `Polynomial.coe_sub`, an error message appearred that lean couldn't recognize which `coe_sub` is in the proof due to ambiguity. Hence I explicitly added `PowerSeries.`.